_posts/2018-11-30-how-to-run-travis-ci-parallel-jobs-with-build-matrix-feature-fast.md

```diff
@@ -54,7 +54,7 @@ env:
     - KNAPSACK_PRO_CI_NODE_INDEX=1
 {% endhighlight %}
 
-How to run Cypress tests in JavaScript on parallel jobs with [@knapsack-pro/cypress](https://github.com/KnapsackPro/knapsack-pro-cypress):
+How to run Cypress tests in JavaScript on concurrent jobs with [@knapsack-pro/cypress](https://github.com/KnapsackPro/knapsack-pro-cypress):
```

Comment:
what's the intention of use concurrent word? Just to have a new keyword in the article? I guess that's fine.
This reminded me about this what's the difference between concurrent and parallel https://softwareengineering.stackexchange.com/a/190725

Reply:
You are right it's technically not the same thing. I've seen there are some visits from a keyword like `travis concurrent jobs`, so I thought we can add this word here and there to improve the ranking on this specific keyword. The text may also read better when you don't duplicate the same word that much, but I'm fine with reverting this to keep it more technically accurate.

Reply:
Let's keep concurrent. :)